manual.typ

```diff
@@ -1415,3 +1374,187 @@ tree.tree(data, content: (padding: .1), direction: "right",
 A tree node is an array of nodes. The first array item represents the
 current node, all following items are direct children of that node.
 The node itselfes can be ot type `content` or `dictionary` with a key `content`.
+
+== Plot
+
+The library `plot` of CeTZ allows plotting 2D data as linechart.
+
+=== Plot <plot-function>
+
+The `plot` function is an environment for plotting data.
+
+Note that different axis-styles can show different axes. The "school-book" style shows only axis "x" and "y", while the "scientific" style can show "x2" and "y2", if set (if unset, "x2" mirrors "x" and "y2" mirrors "y"). Other axes (e.G. "my-axis") work, but no ticks or labels will be shown.
+
+```typc
+plot(size: (width, height),
+     axis-style: "scientific"
+     ..options,
+     body)
+```
+
+The ticks option dictionary supports the following keys:
+#def-arg("size", "a",
+  [Size of the plot])
+#def-arg("axis-style", "s?",
+  [Axis style, either "scientific" or "school-book"])
+#def-arg("..options", "any",
+  [Axis options in the form `<axis-name>-<option>`. The possible options are:
+  #box[
+  / label : The axis label
+  / min: Axis min. value
+  / max: Axis max. value
+  / ticks: List of tick values or value/label tuples
+  / tick-step: Tick step or `none`
+  / minor-tick-step: Minor tick step or `none`
+  / decimals: Number of tick label decimal digits
+  / unit: Tick label suffix
+  ]
+])
+#def-arg("body", "..",
+  [Calls of `plot.add(..)`, see <plot-add>])
+
+=== Plot-Add
+
+The `plot.add` function adds plotting data into a plot environment.
+It must be called from insides `plot({..})`.
+
+If used with a callback, the `domain` must be specified!
+
+```typc
+add(domain: auto, hypograph: false, epigraph: false, fill: false,
+    mark: none, mark-size: .2, mark-style: (:), samples: 100,
+    style: (:), axes: ("x", "y"),
+    data)
+```
+
+#def-arg("domain", "a|auto",
+  [Range of x for sampled data])
```

Comment:
What happens when domain is auto? Explain the expected array types, length and each position means